autoVerticalizationAreaRatio: 0.01
borderPointsAutoFilterPixelOffset: 5
cellHeight: 0.335
cellWidth: 0.48
closenessAreaRatio: 0.002
gridImageHorLim: 20
gridImageVerLim: 20
highH: 155
highS: 255
highV: 255
lowARRatioAutoFilter: 0.3
lowAreaRatioAutoFilter: 0.1
lowContourAreaFilter: 50
lowContourLenFilter: 3
lowH: 147
lowS: 0
lowV: 0
minNodeNum: 68
segmentExtractionPixelOffset: 10
trackerCorrectionPixelOffset: 10
trackerInitPixelOffset: 10
upARRatioAutoFilter: 2
upAreaRatioAutoFilter: 3.5
verticalCenter: 0.25
vxConstCmd: 0.05
vyConstCmd: 0.1
vzConstCmd: 0.1
vzaConstCmd: 0.02
